default/python/universe_generation/galaxy.py

```diff
@@ -297,7 +298,7 @@ def stitching_positions(p1, p2):
 
     # Pick a random point in an 2:1 ratio ellipse and then rotate and slide it in between p1 and p2
     p1_p2_theta = acos((p2[0] - p1[0]) / p1_p2_dist)
-    p1_p2_scale = (p1_p2_dist - 2*min_dist) / 2
+    p1_p2_scale = (p1_p2_dist - 2 * min_dist) / 2
```

Comment:
When using whitespace in formulas, it is prefered to group by mathematical order of operations.

PEP 8:
```
Yes:
x = x*2 - 1
hypot2 = x*x + y*y
c = (a+b) * (a-b)
No:
x = x * 2 - 1
hypot2 = x * x + y * y
c = (a + b) * (a - b)
```
https://www.python.org/dev/peps/pep-0008/#other-recommendations